asllib/doc/ASLTypingReference.tex

```diff
@@ -6767,8 +7617,8 @@ \subsection{Prose}
   \item All of the following apply:
   \begin{itemize}
     \item \name\ is not declared in \tenv.
-    \item  \tenv',\tty' are the result of attempting to add \texttt{name} as a subtype of \texttt{sup} and
-    \item  \tenv',\tty' are the result of attempting to add \texttt{name} as a subtype of \texttt{sup} and
+    \item  \tenv',\tty' are the result of attempting to add $\name$ as a subtype of $\sup$ and
+    \item  \tenv',\tty' are the result of attempting to add $\name$ as a subtype of $\sup$ and
```

Comment:
duplicate? 

also attempting, in what sense?

Reply:
I will fix this in the future PR for this chapter.